### PR TITLE
docs: add FAQ page

### PR DESCRIPTION
**Pull request by takeshi-iwanari**
Signed-off-by: takeshi.iwanari <takeshi.iwanari@tier4.jp>

### CODE REVIEW

File: docs/faq/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+### CLI tool doesn't work
+
+In case CLI tool execution fails, please make sure to perform CARET environment settings
+
+```bash
+source /opt/ros/humble/setup.bash
+source ~/ros2_caret_ws/install/local_setup.bash
+
+ros2 caret check_caret_rclcpp --workspace <path-to-workspace>
```
Comment (hsgwa) [inline docs/faq/index.md:20]: @takeshi-iwanari 
Could you add caution for  Anaconda-based Python?
CLI tools doesn't work property in anaconda environment.
https://github.com/tier4/caret/issues/37#issuecomment-1276952560

File: docs/faq/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+- Message flow (`Node_B` -> `Node_C` -> `Node_D`) looks disconnected at `Node_C` four times every five messages. It's because `Node_C` publishes 1 topic while receiving 5 topics from `Node_B`. So 4 topics don't have a corresponding topic to be published to `Node_D`
+- Note: A similar phenomenon will happen even if `Callback c0` is a timer callback
+
+![node_structure_drop](./imgs/node_structure_drop.drawio.png)
+
+![message_flow_split_drop](./imgs/message_flow_drop_graph.png)
+
+### How response time is calculated?
+
+- In general, response time is the time a system or functional unit takes to react to a given input ([reference](<https://en.wikipedia.org/wiki/Response_time_(technology)>)). Response time calculated by CARET is the time it takes for input data to arrive at the last node. It doesn't include processing time at the last node nor latency of an actuator. It's calculated as the sum of callback processing time and communication latency time in a path
```
Comment (hsgwa) [inline docs/faq/index.md:124]: Strictly speaking, path latency is sum of communication latency and node latency.
here, communication latency is publish - subscribe latency, and node latency is subscribe - publish latency.
Though it depends on implementations, callback latency doesn't equal to node latency.
> sum of callback processing time and communication latency

File: docs/faq/index.md (hunk 1 above)
Comment (takeshi-iwanari) [reply docs/faq/index.md:20]: https://github.com/tier4/CARET_doc/pull/89/commits/e9b5d479466fed28efee8c311c25d300fea67490 adds the caution.

**Review by hsgwa — APPROVED** [review]
LGTM!

Thank you for adding FAQ page!

**Pull request merged**